extensions/ql-vscode/src/common/query-language.ts

```diff
@@ -25,7 +25,7 @@ export const PACKS_BY_QUERY_LANGUAGE = {
   [QueryLanguage.Ruby]: ["codeql/ruby-queries"],
 };
 
-export const dbSchemeToLanguage = {
+export const dbSchemeToLanguage: Record<string, string> = {
```

Comment:
Would it be more accurate to say this instead:

```suggestion
export const dbSchemeToLanguage: Record<string, QueryLanguage> = {
```

(you might also need to update the values to say `QueryLanguage.JavaScript` instead of `javascript`)

Reply:
Yep, we can definitely use the `QueryLanguage` enum in more places in this code.